src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -253,7 +255,7 @@ private void intakeCoralFromFloorState() {
       m_gripper.setAlgaeGripSpeed(GripperConstants.IntakeCoralOnAlgaeMotorSpeed);
     } else if (m_gripper.hasCoralFront() && !m_gripper.hasCoralBack()) {
       m_gripper.setCoralGripSpeed(GripperConstants.IntakeCoralSlow);
-      m_gripper.setAlgaeGripSpeed(0.0);
+      m_gripper.setAlgaeGripSpeed(GripperConstants.IntakeCoralOnAlgaeMotorSpeed);
```

Comment:
0.0 wasn't good enough, but the full speed might be too much. We might want to try some compromised value.